packages/cli/src/templates/package.ts

```diff
@@ -0,0 +1,45 @@
+type PackageTemplateOptions = {
+  name: string;
+  spectacleVersion: string;
+  usesTypeScript: boolean;
+};
+
+export const packageTemplate = (options: PackageTemplateOptions) =>
+  `{
+  "name": "${options.name}",
+  "private": true,
+  "scripts": {
+    "start": "webpack-dev-server --hot --config ./webpack.config.js",
+    "clean": "rimraf dist",
+    "build": "webpack --config ./webpack.config.js --mode production"
+  },
+  "dependencies": {
+    "spectacle": "${options.spectacleVersion}",
```

Comment:
Super minor, but do you think including `^` in the version number feels more appropriate?